+nix/Block.m

```diff
@@ -1,5 +1,9 @@
-classdef Block < nix.Entity
+classdef Block < nix.NamedEntity
     %Block nix Block object
+    properties (Access = protected)
+        dynamic_attrs = {'dataArrayCount', 'sourceCount', 'tagCount', ...
```

Comment:
seems they are still there.

Actually it might be useful in some case. We made some profiling recently and found that the nix-mx operations are not very fast. So in (maybe not very realistic) case when one has 1000 or more blocks and wants to just get an overview how many data arrays are in the file one could just sum the 'dataArrayCount' without fetching all data arrays, which should be faster. However it's a question if we should accommodate this case at all.